views/scan.hbs

```diff
@@ -1,97 +1,32 @@
-          </div>
-        </div>
-      </form>
-      <p class="thin">
-        Your email will not be stored or sent content. Find out more in our Privacy Policy.
-      </p>
-      <p>
-        Get an alert if your account information is compromised.<br>
-        <a>Subscribe to Firefox Monitor</a>
-      </p>
-    </div>
-  </div>
-
-  <div class="cell small-1"></div>
-
-
-{{#if foundBreaches }}
-  <div class="cell"></div>
-
-  <div class="cell small-1"></div>
-
-  <div class="cell small-6 rounded-cell flexbox grid-x grid-padding-x grid-padding-y no-padding">
-    <div class="cell tips-header bottom-border">
-      <div class="text-center">
-        <h4 class="bold">Your Compromised Accounts</h4>
       </div>
-    </div>
-    {{#each foundBreaches }}
-      <div class="cell flexbox bottom-border">
-        <div class="protect-img breach-img">
-          <img src="img/{{ name }}.png">
-        </div>
-        <div class="flex">
-          <h4>{{ name }}</h4><br>
-          <p>
-            <span class="bold">Breach date:</span> {{prettyDate meta.BreachDate }}<br>
-            <span class="bold">Compromised data:</span> {{breachDataClasses meta.DataClasses }}<br>
-            <span class="bold">Compromised accounts:</span> {{localeString meta.PwnCount }}<br>
-            {{! TODO: make sure meta.Description is a safe string so we can use triple-{{{ }}
-            {{{ meta.Description }}}
-          </p>
-        </div>
+    </div><!--closes banner-left-->
+    <div id="banner-right">
+      <img id="arrow-divider" src="img/arrow_divider.png" />
+      <div class="banner-content">
+        {{> sign_up }}
       </div>
-    {{/each}}
-    <div class="cell"></div>
+    </div><!--close banner-right -->
   </div>
+</div>
 
-  <div class="cell small-4 rounded-cell flexbox grid-y grid-padding-x grid-padding-y no-padding">
-    <div class="cell tips-header bottom-border">
-      <div class="text-center">
-        <h4 class="bold">What to Do</h4>
-      </div>
-    </div>
-    <div class="cell">
-      <p><span class="bold">Change your password</span> on these sites and anywhere else you’ve used the same password.</p>
-      <p>Make your answers to <span class="bold">security questions</span> just as strong as your passwords.</p>
-      <p><span class="bold">Use password managers</span> like 1Password, LastPass, or Dashlane to generate strong passwords for you, remember them for you, and fill them into websites so you don’t have type them in.</p>
-      <p>Sign up for alerts from Firefox Monitor to learn sooner about your compromised accounts.</p>
-    </div>
-    <div class="cell"></div>
+{{#if foundBreaches }}
+  <div class="section-container">
+    <div id="compromised-accounts-wrapper">
+    <div class="column-container-twothirds">{{> compromised_accounts }}</div>
+    <div class="column-container-third">{{> what_to_do }}</div>
+  </div>
+  {{ else }}
+    {{> tips }}
   </div>
```

Comment:
I think this needs to be moved up into the `{{#if}}` block above, as it looks like it has unbalanced `<div>..</div>` tags.